.github/workflows/build.yaml

```diff
@@ -0,0 +1,40 @@
+name: build
```

Comment:
I would change the name to `ci` or something similar.

Reply:
Done